engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-subscriptions-modal.html

```diff
@@ -80,7 +80,7 @@ <h4 data-block="modal-header" translate>
             <tr bst-table-row ng-repeat-start="(name, subscriptions) in groupedSubscriptions">
               <td class="row-select"></td>
               <td colspan="8">
-                <a ui-sref="subscription.info({subscriptionId: subscription.id})" class="confined-text">
+                <a ng-click="ok();"  ui-sref="subscription.info({subscriptionId: subscription.id})" class="confined-text">
```

Comment:
This confirmation modal doesn't lock out the yes/no buttons after the first click, so I can easily get 500 errors from the server by clicking yes/no multiple times before I'm redirected to the task progress.

Reply:
This probably should be made into a separate issue rather than being fixed in this PR.

Reply:
I will go ahead open an issue around the same.

The transitionTo() method was used to redirect to the errata details and content hosts page specific to the errata action on host collections. 
I am trying not to use those page as redirect links but content inside the existing modal seems more user friendly instead of the redirection.